### PR TITLE
Graphical components for responses in Reactor Board application

### PR DESCRIPTION
**Pull request by tkurylek**
issue #28 

For now every possible type that can be rendered by `JSONReactorResponseRenderer` is handled.
reactor-tk-28

### CODE REVIEW

File: reactor-lib/reactor-transport-http-jetty/src/main/java/org/reactor/transport/http/rest/RestHandler.java [hunk 1]
```diff
@@ -39,11 +48,19 @@ private void processTransportMessage(String reactorInput, HttpServletResponse re
 
         PrintWriter writer = response.getWriter();
         ReactorResponseRenderer renderer = new JSONReactorResponseRenderer();
-        requestHandler.handleReactorRequest(new ReactorRequestInput(reactorInput), SENDER, renderer);
+        awaitResponse(requestHandler.handleReactorRequest(new ReactorRequestInput(reactorInput), SENDER, renderer));
         renderer.commit(writer);
         writer.flush();
     }
 
+    private void awaitResponse(Future<?> request) {
```
Comment (activey) [inline reactor-lib/reactor-transport-http-jetty/src/main/java/org/reactor/transport/http/rest/RestHandler.java]: nice :)

File: reactor-lib/reactor-transport-http-jetty/src/main/resources/static/board/js/directives/directive-response-renderer.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<div data-ng-repeat="(key, value) in content">
```
Comment (activey) [inline reactor-lib/reactor-transport-http-jetty/src/main/resources/static/board/js/directives/directive-response-renderer.html]: thats it :))

File: pom.xml [hunk 3]
```diff
@@ -211,5 +211,6 @@
         <module>reactor-lib</module>
 
         <module>travelling</module>
+        <module>reactor-renderer-json</module>
```
Comment (activey) [inline pom.xml]: please move it a little higher just below reactor-lib

File: reactor-lib/reactor-transport-http-jetty/src/main/java/org/reactor/transport/http/rest/RestHandler.java [hunk 4]
```diff
@@ -14,11 +17,17 @@
...
 import org.reactor.response.renderer.ReactorResponseRenderer;
 import org.reactor.transport.ReactorRequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class RestHandler extends AbstractHandler {
 
+    private static final Logger LOG = LoggerFactory.getLogger(RestHandler.class);
```
Comment (activey) [inline reactor-lib/reactor-transport-http-jetty/src/main/java/org/reactor/transport/http/rest/RestHandler.java]: please change name to LOGGER

Comment (tkurylek) [inline reactor-lib/reactor-transport-http-jetty/src/main/java/org/reactor/transport/http/rest/RestHandler.java]: Not a problem :)

File: pom.xml (hunk 3 above)
Comment (tkurylek) [inline pom.xml]: Yeah, I'll do that.

**Pull request merged**